### ISSUE TITLE
page title fails color contrast

### ISSUE THREAD

**Issue by philwilliammee** [CU-CommunityApps/cd-son#66]
The title on the page fails color contrast.

![image](https://user-images.githubusercontent.com/4685094/80100272-bd152400-853d-11ea-976e-f01365de137d.png)
